test/test_local_db.js

```diff
@@ -261,36 +264,8 @@ var testReplicaIncorrectPassword = function (cb) {
 }
 
 var testparseMongoConnectionString = function (cb) {
```

Comment:
Should we try keep the tests consistent in structure? Maybe export the `goodStrings` tests too